boards/esp32s2-wemos-mini/Makefile.include

```diff
@@ -0,0 +1,6 @@
+PORT_LINUX ?= /dev/ttyACM0
+
+USB_VID ?= $(USB_VID_TESTING)
+USB_PID ?= $(USB_PID_TESTING)
+
+include $(RIOTBOARD)/common/esp32s2/Makefile.include
```

Comment:
Please add the following, otherwise the 2nd stage bootloader output (`esp_log_startup` is used or `LOG_LEVEL` > 3) is printed to UART0 default pins GPIO43.

```suggestion
CFLAGS += -DCONFIG_CONSOLE_UART_TX=39
CFLAGS += -DCONFIG_CONSOLE_UART_RX=37

include $(RIOTBOARD)/common/esp32s2/Makefile.include
```